llvm/utils/gn/secondary/bolt/unittests/Core/BUILD.gn

```diff
@@ -15,6 +15,7 @@ unittest("CoreTests") {
     "BinaryContext.cpp",
     "DynoStats.cpp",
     "MCPlusBuilder.cpp",
+    "MemoryMaps.cpp",
```

Comment:
(No need to change .gn files; they're updated automatically by a bot when possible.)

Reply:
Good to know. Thanks!